### DISCUSSION TITLE
this thing its not working anymore after updating.

### DISCUSSION THREAD

**Discussion by elen07zz** [vladmandic/sdnext#708] [General]
╭────────────────────────────────────────────────────────────────────── Traceback (most recent call last) ──────────────────────────────────────────────────────────────────────╮
│ /home/akairax/automatic/launch.py:104 in <module>                                                                                                                             │
│                                                                                                                                                                               │
│   103         exit(0)                                                                                                                                                         │
│ ❱ 104     import webui                                                                                                                                                        │
│   105     webui.webui()                                                                                                                                                       │
│                                                                                                                                                                               │
│ /home/akairax/automatic/webui.py:35 in <module>                                                                                                                               │
│                                                                                                                                                                               │
│    34 import ldm.modules.encoders.modules # pylint: disable=W0611,C0411                                                                                                       │
│ ❱  35 from modules import extra_networks, ui_extra_networks_checkpoints # pylint: disable=C041                                                                                │
│    36 from modules import extra_networks_hypernet, ui_extra_networks_hypernets, ui_extra_netwo                                                                                │
│                                                                                                                                                                               │
│ /home/akairax/automatic/modules/ui_extra_networks_checkpoints.py:5 in <module>                                                                                                │
│                                                                                                                                                                               │
│    4                                                                                                                                                                          │
│ ❱  5 from modules import shared, ui_extra_networks, sd_models                                                                                                                 │
│    6                                                                                                                                                                          │
│                                                                                                                                                                               │
│ /home/akairax/automatic/modules/ui_extra_networks.py:11 in <module>                                                                                                           │
│                                                                                                                                                                               │
│    10 from modules import shared                                                                                                                                              │
│ ❱  11 from modules.images import read_info_from_image                                                                                                                         │
│    12 from modules.generation_parameters_copypaste import image_from_url_text                                                                                                 │
│                                                                                                                                                                               │
│ /home/akairax/automatic/modules/images.py:16 in <module>                                                                                                                      │
│                                                                                                                                                                               │
│    15                                                                                                                                                                         │
│ ❱  16 from modules import sd_samplers, shared, script_callbacks, errors                                                                                                       │
│    17 from modules.shared import opts, log                                                                                                                                    │
│                                                                                                                                                                               │
│ /home/akairax/automatic/modules/sd_samplers.py:1 in <module>                                                                                                                  │
│                                                                                                                                                                               │
│ ❱  1 from modules import sd_samplers_compvis, sd_samplers_kdiffusion, shared                                                                                                  │
│    2                                                                                                                                                                          │
│                                                                                                                                                                               │
│ /home/akairax/automatic/modules/sd_samplers_kdiffusion.py:8 in <module>                                                                                                       │
│                                                                                                                                                                               │
│     7     pass                                                                                                                                                                │
│ ❱   8 import k_diffusion.sampling                                                                                                                                             │
│     9 from modules import prompt_parser, devices, sd_samplers_common                                                                                                          │
│                                                                                                                                                                               │
│ /home/akairax/automatic/repositories/k-diffusion/k_diffusion/__init__.py:1 in <module>                                                                                        │
│                                                                                                                                                                               │
│ ❱ 1 from . import augmentation, config, evaluation, external, gns, layers, models, sampling,                                                                                  │
│   2 from .layers import Denoiser                                                                                                                                              │
│                                                                                                                                                                               │
│ /home/akairax/automatic/repositories/k-diffusion/k_diffusion/evaluation.py:6 in <module>                                                                                      │
│                                                                                                                                                                               │
│     5 from cleanfid.inception_torchscript import InceptionV3W                                                                                                                 │
│ ❱   6 import clip                                                                                                                                                             │
│     7 from resize_right import resize                                                                                                                                         │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
ModuleNotFoundError: No module named 'clip'

**Comment by elen07zz** [comment]
then running with --upgrade

13:43:42-078655 INFO     Extensions enabled: ['clip-interrogator-ext', 'prompt-bracket-checker', 'LDSR', 'sd-webui-controlnet', 'sd-webui-model-converter',       
                         'stable-diffusion-webui-images-browser', 'sd-extension-system-info', 'sd-dynamic-thresholding',                                          
                         'multidiffusion-upscaler-for-automatic1111', 'stable-diffusion-webui-rembg', 'ScuNET', 'Lora', 'seed_travel', 'SwinIR']                  
13:43:42-676070 ERROR    Error running extension installer: /home/akairax/automatic/extensions-builtin/clip-interrogator-ext/install.py                           
13:43:44-339839 ERROR    Error running extension installer: /home/akairax/automatic/extensions-builtin/stable-diffusion-webui-images-browser/install.py           
13:43:46-560707 ERROR    Error running extension installer: /home/akairax/automatic/extensions-builtin/stable-diffusion-webui-rembg/install.py                    
13:43:47-226805 ERROR    Error running extension installer: /home/akairax/automatic/extensions-builtin/seed_travel/install.py

**Comment by elen07zz** [comment]
╭────────────────────────────────────────────────────────────── Traceback (most recent call last) ───────────────────────────────────────────────────────────────╮
│ /home/akairax/automatic/launch.py:105 in <module>                                                                                                              │
│                                                                                                                                                                │
│   104     import webui                                                                                                                                         │
│ ❱ 105     webui.webui()                                                                                                                                        │
│   106                                                                                                                                                          │
│                                                                                                                                                                │
│ /home/akairax/automatic/webui.py:243 in webui                                                                                                                  │
│                                                                                                                                                                │
│   242     log.debug('Entering WebUI')                                                                                                                          │
│ ❱ 243     start_ui()                                                                                                                                           │
│   244     load_model()                                                                                                                                         │
│                                                                                                                                                                │
│ /home/akairax/automatic/webui.py:192 in start_ui                                                                                                               │
│                                                                                                                                                                │
│   191     async_policy()                                                                                                                                       │
│ ❱ 192     initialize()                                                                                                                                         │
│   193     if shared.opts.clean_temp_dir_at_start:                                                                                                              │
│                                                                                                                                                                │
│ /home/akairax/automatic/webui.py:93 in initialize                                                                                                              │
│                                                                                                                                                                │
│    92     modelloader.cleanup_models()                                                                                                                         │
│ ❱  93     modules.sd_models.setup_model()                                                                                                                      │
│    94     startup_timer.record("models")                                                                                                                       │
│                                                                                                                                                                │
│ /home/akairax/automatic/modules/sd_models.py:90 in setup_model                                                                                                 │
│                                                                                                                                                                │
│    89         os.makedirs(model_path)                                                                                                                          │
│ ❱  90     list_models()                                                                                                                                        │
│    91     enable_midas_autodownload()                                                                                                                          │
│                                                                                                                                                                │
│ /home/akairax/automatic/modules/sd_models.py:113 in list_models                                                                                                │
│                                                                                                                                                                │
│   112     elif shared.cmd_opts.ckpt != shared.default_sd_model_file:                                                                                           │
│ ❱ 113         shared.log.warning(f"Checkpoint not found: {shared.cmd_opts.ckpt}", file=sys.std                                                                 │
│   114     for filename in sorted(model_list, key=str.lower):                                                                                                   │
│                                                                                                                                                                │
│ /usr/lib/python3.10/logging/__init__.py:1489 in warning                                                                                                        │
│                                                                                                                                                                │
│   1488         if self.isEnabledFor(WARNING):                                                                                                                  │
│ ❱ 1489             self._log(WARNING, msg, args, **kwargs)                                                                                                     │
│   1490                                                                                                                                                         │
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
TypeError: Logger._log() got an unexpected keyword argument 'file'

**Comment by SoraEnkai** [comment]
This is already mentioned in issues: https://github.com/vladmandic/automatic/issues/709

**Comment by vladmandic** [comment]
and fixed. sorry. amazing community - this is 5th issue for this in matter of minutes after bad push.
